### DISCUSSION TITLE
Crashes occur frequently, CefSharp.BrowserSubprocess.exe crash in 0xe0000008

### DISCUSSION THREAD

**Discussion by Cricle** [cefsharp/CefSharp#4083] [General]
|Name|Status|
|:-:|:-:|
|.NET Runtime|.NET Framework 4.5.2|
|LAA| Enable |
|CefSharp version|  81.3.100 |
|OS| win10 |
|EChart verrsion|5.2.2|
|Configuration|x86|

It will crash in a few hours after running. 
Sometimes it crashes with the host program.

Cef settign is here

```csharp
settings.SetOffScreenRenderingBestPerformanceArgs();
settings.LogSeverity = LogSeverity.Verbose;
```

![w](https://user-images.githubusercontent.com/26201412/165890132-4a0dec93-81c7-47cd-b68d-8764dbcaa562.png)

The application behavior is to refresh the chart once a second, but there's nothing wrong with using the web.

Many times before ` CefSharp.BrowserSubprocess.exe ` will throw `SEHException` and crash.

Already flollow wiki enable LAA, but many crash time, app and CefSharp.BrowserSubprocess.exe no more than 1G of memory.

**Comment by Cricle** [comment]
Windbg show 

![image](https://user-images.githubusercontent.com/26201412/165894678-77b0b6eb-e811-493f-9bb2-6206e2f713f8.png)

**Reply by amaitland** [reply]
> CefSharp version | 81.3.100

Converting to discussion as this is an old unsupported version. Please only open an issue if the problem reproduces with the current supported version (version 100 at time of writing).

You should get a crash dump see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#crash-reporting

If you suspect that your application is running out of memory I'd suggest testing the 64bit version.

**Reply by Cricle** [reply]
Thanks. I will try  some method

**Reply by amaitland** [reply]
Once you have a crash dump see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#loading-native-symbols-for-easier-diagnosis for details on downloading libcef.dll.pdb 

Once you get a stack trace you'll know more.

**Comment by ljhaust** [comment]
@Cricle Do you solve this problem ? It is happen to me.But i can not solve it.

**Reply by amaitland** [reply]
Upgrade to the latest supported version.

If the problem still persists get a crash dump, load the symbols, get a stack trace. 
https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#crash-reporting
